apps/lib/publisher_agent/config.go

```diff
@@ -251,6 +256,38 @@ func LoadConfig(configFilePath string, keysFilePath string) (*StorkPublisherAgen
 	return config, nil
 }
 
+// this overwrites any existing values in keysFile
+func LoadKeysFileFromEnv(keysFile *KeysFile) {
```

Comment:
This prob doesn't need to be capitalized - capitalization is akin to marking a fn as public for the module